src/ui/ChewingEditor.ui

```diff
@@ -43,16 +43,23 @@
      </widget>
     </item>
     <item row="0" column="3">
+     <widget class="QPushButton" name="undoButton">
+      <property name="text">
+       <string>Undo</string>
+      </property>
+     </widget>
+    </item>
+    <item row="0" column="4">
```

Comment:
just an idea, is it better to put this button on the right (next to) `Remove` button?